.travis.yml

```diff
@@ -37,6 +37,8 @@ before_script:
 
 script:
   - ./phpunit --coverage-clover=coverage.xml
+  - phpenv config-rm xdebug.ini; true
+  - phpdbg -qrr ./phpunit --coverage-text
```

Comment:
What is the point of running the tests once collecting code coverage data using Xdebug and once collecting code coverage data using PHPDBG?

Reply:
It was just the quickest way I came up with for testing the code coverage through phpdbg use case and ensuring all the tests passed through the sapi. I can certainly take it out if you think it doesn't provide any future value.

Reply:
Yes, please.